package.json

```diff
@@ -61,8 +50,19 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
     "@typescript-eslint/eslint-plugin": "^3.5.0",
     "@typescript-eslint/parser": "^3.5.0",
-    "eslint-plugin-react": "^7.20.2"
+    "eslint-plugin-react": "^7.20.2",
+    "node-sass": "^4.14.1",
+    "prop-types": "^15.7.2",
+    "react-scripts": "3.4.1",
+    "sass": "^1.26.10"
```

Comment:
I also don't know why sass and node-sass are installed. I don't see any sass or scss files in the repository, so we might be able to delete this too?